daemon/config_unix.go

```diff
@@ -89,6 +95,7 @@ func (config *Config) InstallFlags(flags *pflag.FlagSet) {
 	flags.Int64Var(&config.CPURealtimePeriod, "cpu-rt-period", 0, "Limit the CPU real-time period in microseconds")
 	flags.Int64Var(&config.CPURealtimeRuntime, "cpu-rt-runtime", 0, "Limit the CPU real-time runtime in microseconds")
 	flags.StringVar(&config.SeccompProfile, "seccomp-profile", "", "Path to seccomp profile")
+	flags.Var(&config.ShmSize, "default-shm-size", "Set the default shm size for containers")
```

Comment:
Perhaps `Default shm size for container` to be more consistent with other options